lrc/src/Pos/Lrc/DB/Richmen.hs

```diff
@@ -23,27 +23,34 @@ module Pos.Lrc.DB.Richmen
 
```

Comment:
Wow, you could move a module from one package to another and the name was correct!!!